src/c20_server/documentsIDs.py

```diff
@@ -0,0 +1,35 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+def get_documents_ids(api_url, api_key, docket_id):
```

Comment:
The api_url should not be a parameter., just the key and docket_id

Reply:
I am going to fix that